test/libsolidity/syntaxTests/functionTypes/external_function_type_to_address.sol

```diff
@@ -1,5 +1,5 @@
 contract C {
     function f() public view returns (address) {
-        return address(this.f);
```

Comment:
Can you also add the old test to see that the conversion does not work anymore? Also, we should provide a hint about how to change the code in the error message (if possible).